command/ibft/ibft_candidates.go

```diff
@@ -0,0 +1,94 @@
+package ibft
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-sdk/command/helper"
+	ibftOp "github.com/0xPolygon/polygon-sdk/consensus/ibft/proto"
+	empty "google.golang.org/protobuf/types/known/emptypb"
+)
+
+// IbftCandidates is the command to query the snapshot
+type IbftCandidates struct {
+	helper.Meta
+}
+
+// GetHelperText returns a simple description of the command
+func (p *IbftCandidates) GetHelperText() string {
+	return "Queries the current set of proposed candidates, as well as candidates that have not been included yet"
+}
+
+func (p *IbftCandidates) GetBaseCommand() string {
+	return "ibft candidates"
+}
+
+// Help implements the cli.IbftCandidates interface
```

Comment:
This comment seems out of context. Same for the other comment references to cli.IbftCandidates interface in the file

Reply:
These are just generic command descriptions we have for each method commands need to implement